source/index.md.erb

```diff
@@ -106,30 +106,30 @@ The API provides access to the following data collections in your account via RE
 
 # Webhook Support
 
-The 10,000ft API supports webhooks for certain events. See [the webhooks section](#webhooks) for details.
+The Resource Management by Smartsheet API supports webhooks for certain events. See [the webhooks section](#webhooks) for details.
 
 # Zapier Integration
 
-Our Zapier integration helps you integrating with the 10,000ft API in convenient and powerful ways and connect many other apps and services used by yor team.
+Our Zapier integration helps you integrating with the Resource Management by Smartsheet API in convenient and powerful ways and connect many other apps and services used by yor team.
 
-* [10,000ft on Zapier](https://zapier.com/apps/10000ft/integrations)
+* [Resource Management by Smartsheet on Zapier](https://zapier.com/apps/10000ft/integrations)
 * [Getting started with Zapier](#zapier-integration)
 
 <%# # Case studies
 
-Some case studies of our customers using our API to build new tools and applications using the 10Kft API.
+Some case studies of our customers using our API to build new tools and applications using the Resource Management by Smartsheet API.
 
   * [Fancy Pants Group](https://www.10000ft.com/blog/fancy-pants-group-case-study), a digital creative studio based in New York, recently used our API to develop and launch a custom project importer cheekily called Job the Builder. Their goal is to make it easy for everyone on their 100-person team to see and communicate consistent data across multiple tools. %>
 
 # Third party libraries
 
-A few of our customers have implemented and shared API client libraries to help using the 10Kft API easier. Depending on your scenario, you may benefit from adopting and improving on them. Here are a couple of references,
+A few of our customers have implemented and shared API client libraries to help using the Resource Management by Smartsheet API easier. Depending on your scenario, you may benefit from adopting and improving on them. Here are a couple of references,
 
 * Revelry Labs / `ruby` / [https://github.com/revelrylabs/tenk](https://github.com/revelrylabs/tenk)
 * Spindance / `ruby` / [https://github.com/spindance/ten_thousand_feet](https://github.com/spindance/ten_thousand_feet)
 
 # Questions?
 
-Questions or feedback about our API, please submit a support ticket at our [Support Page](https://help.smartsheet.com/contact/10000ft)
+Questions or feedback about our API, please submit a support ticket at our [Support Page](https://help.smartsheet.com/contact/resource-management)
```

Comment:
Do we need to do anything here or other team need to change it? The new url are not be able to reach @aiyer-smartsheet

Reply:
There is a team working on enabling the new Resource management pages now. We should have this live by eod.

Reply:
@baonguyen84 This link will be working soon. There is a team getting it up. It's the same link in prod.